index.html

```diff
@@ -102,18 +101,18 @@ <h6>Registration now Open!</h6>
 					  <div class="collapsible-content">
 					    <div class="content-inner">
 					      <p>
-                                Our Mini-Hackathon is an event where students from across the Atlanta Metropolitan Area and beyond will come together for a day of innovation and creativity. Participants have 10 hours to create anything that shows off their creativity and passion for development. You may choose any platform, programming language, or format to show your stuff. You can even present a storyboard or idea; there’s no end to the possibilities!			  
+                                Hackathons are events where students from across the country come together for a weekend of innovation and creativity. Participants have 36 hours to create anything that shows off their creativity and passion for development. You may choose any platform, programming language, or format to show your stuff. You can even present a storyboard or idea; there’s no end to the possibilities!			  

```

Comment:
Abandon these changes, keep the code from the most recent version on the master branch. This goes for all other FAQs as well